src/UI/InputField/InputField.tsx

```diff
@@ -1,31 +1,70 @@
+import { useState } from 'react';
+import EyeIcon from '../../assets/icons/IconsSvg/EyeIcon';
+import EyeIconHidden from '../../assets/icons/IconsSvg/EyeIconHidden';
 import './InputField.css';
 
-interface InputFieldType {
-  type: string;
-  id: string;
-  name: string;
-  minLength: number;
-  required: boolean;
+interface InputFieldProps {
```

Comment:
Use style name like "IInputFieldProps" it's better. All interface start from short "I...."